stable/moodle/values.yaml

```diff
@@ -70,18 +127,42 @@ persistence:
     # storageClass:
     accessMode: ReadWriteOnce
     size: 1Gi
+    #existingClaim: ""
   moodle:
     ## If defined, volume.beta.kubernetes.io/storage-class: <storageClass>
     ## Default: volume.alpha.kubernetes.io/storage-class: default
     ##
     # storageClass:
     accessMode: ReadWriteOnce
     size: 8Gi
+    #existingClaim: ""
+
+## Define affinity for the moodle pod
+## Sometimes required when persistent volumes are defined externally
+## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector    
+#affinity:
+#  nodeAffinity:
+#    requiredDuringSchedulingIgnoredDuringExecution:
+#      nodeSelectorTerms:
+#      - matchExpressions:
+#        - key: node-role.kubernetes.io/master
+#          operator: Exists
+#          values:
+#          - machine01
+#    preferredDuringSchedulingIgnoredDuringExecution:
+#    - weight: 1
+#      preference:
+#        matchExpressions:
+#        - key: another-node-label-key
+#          operator: In
+#          values:
+#          - another-node-label-value
 
 ## Configure resource requests and limits
 ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
 ##
 resources:
   requests:
-    memory: 512Mi
-    cpu: 300m
+    memory: 768Mi
+    cpu: 600m
```

Comment:
Do you really need to bump this setting? With this change I am no longer able to deploy the chart on a 3 node GKE cluster.